ckanext/dia_theme/templates/home/snippets/search.html

```diff
@@ -15,7 +15,7 @@ <h3 class="heading"><label for="searchbox">{{ _("Search data") }}</label></h3>
   <div class="tags">
     <h3>{{ _('Popular tags') }}</h3>
     {% for tag in tags %}
-      <a class="tag" href="{% url_for controller='dataset' if h.check_ckan_version('2.9') else 'package', action='search', tags=tag.name %}">{{ h.truncate(tag.display_name, 22) }}</a>
+      <a class="tag" href="{% url_for controller='dataset', action='search', tags=tag.name %}">{{ tag.display_name|truncate(22) }}</a>
```

Comment:
New template syntax for helpers in CKAN 2.10, piping values to helpers instead of accessing them as a function on the global `h` property.